Makefile

```diff
@@ -39,6 +39,9 @@ lint:
 lint-markdown:
 	markdownlint --ignore documentation/node_modules --dot .
 
+lint-markdown-fix:
```

Comment:
I've always like used the pattern above this with lint. In this case `lint-markdown` is what the user uses and the CI becomes

```
golangci-lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4

      - uses: actions/setup-go@v5
        with:
          go-version: ${{ env.GO_VERSION }}

      - run: make lint FIX=""
```

something like the above

Reply:
mmmh I see, I'm removing it then